src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,14 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length ? (
+      comments.map(comment => (
+        <CommentInfo key={comment.id} comment={comment} />
+      ))
+    ) : (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    )}
```

Comment:
Move this check to upper lvl instead render this component
```suggestion
      {comments.map(comment => (
        <CommentInfo key={comment.id} comment={comment} />
      ))}
```